common/app/conf/switches/ServerSideExperimentSwitches.scala

```diff
@@ -1,21 +1,15 @@
 package conf.switches
 
 import conf.switches.Expiry.never
-import experiments.ActiveExperiments
 
 trait ServerSideExperimentSwitches {
-  val ServerSideExperiments = {
-    // It's for the side effect. Blame agents.
-    val experiments = ActiveExperiments.allExperiments
```

Comment:
we don't need this any more? the comment worries me

Reply:
I put it back with a better comment